vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/model/SequenceActionType.java

```diff
@@ -129,6 +132,27 @@ public Type getArgumentType()
             return argumentType;
         }
     },
+    PRESS_KEYS_SIMULTANEOUSLY(false)
+    {
+        private final Type argumentType = TypeUtils.parameterize(List.class, String.class);
+
+        @Override
+        public void addAction(Actions actions, Object argument)
+        {
+            perform(argument, (List<String> arg) -> {
+                LinkedList<String> linkedArg = new LinkedList<>(arg);
+                String firstKey = linkedArg.pollFirst();
+                Keys pressedKey = EnumUtils.isValidEnum(Keys.class, firstKey) ? Keys.valueOf(firstKey) : null;
+                actions.keyDown(pressedKey).sendKeys(KeysUtils.keysToCharSequenceArray(linkedArg)).keyUp(pressedKey);
```

Comment:
passing null as pressedKey will lead to IllegalArgumentException or NPE as per Actions logic

Reply:
Yes. I try to keep the logic that wrong keys order should abort the method and throws the runtime exception

Reply:
refactored